src/DepartmentService.Data/DepartmentUserRepository.cs

```diff
@@ -191,18 +212,27 @@ public Task<List<DbDepartmentUser>> GetAsync(List<Guid> usersIds, bool includeDe
 
     public Task<List<DbDepartmentUser>> GetAsync(IGetDepartmentsUsersRequest request)
     {
-      IQueryable<DbDepartmentUser> dbDepartmentUser = request.ByEntryDate.HasValue 
+      IQueryable<DbDepartmentUser> departmentUsersQuery = request.ByEntryDate.HasValue 
         ? _provider.DepartmentsUsers
             .TemporalBetween(
               request.ByEntryDate.Value,
               request.ByEntryDate.Value.AddMonths(1))
-            .Where(u => u.IsActive).Distinct()
-            .AsQueryable()
-        : _provider.DepartmentsUsers.AsQueryable().Where(du => du.IsActive);
+        : _provider.DepartmentsUsers.AsQueryable();
 
-      dbDepartmentUser = dbDepartmentUser.Where(du => request.DepartmentsIds.Contains(du.DepartmentId));
+      departmentUsersQuery = departmentUsersQuery.Where(du => request.DepartmentsIds.Contains(du.DepartmentId));
+
+      if (request.IncludePendingUsers)
+      {
+        departmentUsersQuery = from users in departmentUsersQuery.Where(du => du.IsActive || du.IsPending)

```

Comment:
Why are you using native sql, and nit LINQ? For optimization or readable?

Reply:
for readable, sql requests are same

Reply:
> Why are you using native sql, and not LINQ? For optimization or readable?

Actually that's is not native SQL, it's just another syntax of LINQ for those who prefer sql-like code